src/views/Sidebar.js

```diff
@@ -0,0 +1,57 @@
+/**
```

Comment:
Love how clean this file ended up by moving the pieces of logic to the controller, etc.